azkaban-common/src/main/java/azkaban/executor/OnContainerizedExecutionEventListener.java

```diff
@@ -7,8 +7,10 @@
 import azkaban.DispatchMethod;
 import azkaban.flow.Flow;
 import azkaban.flow.FlowUtils;
+import azkaban.metrics.MetricsManager;
 import azkaban.project.Project;
 import azkaban.project.ProjectManager;
+import com.codahale.metrics.Counter;
```

Comment:
I am wondering what this dependency is; if we use this Counter, how can we get the metrics value then?
If we want to integrate with InGraphs then we might need to use the internal libraries for this so the value will be reflected on InGraphs.